### DISCUSSION TITLE
How to create Message Class from Message Properties

### DISCUSSION THREAD

**Discussion by rnveach** [checkstyle/eclipse-cs#470] [Q&A]
I am new to Eclipse plugin development,

How do you create the message class (https://github.com/checkstyle/eclipse-cs/blob/master/net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/Messages.java) from the property file (https://github.com/checkstyle/eclipse-cs/blob/master/net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/messages.properties).

I assume they are somehow auto-generated and that is why Checkstyle is disabled for them.

**Comment by Calixte** [comment]
I think you can right-click on some class, and select "External Strings" to generate both the `messages.properties` file and the `Messages.java` class. Strings with the comment `//$NON-NLS-1$` will be ignored.
